src/style.css

```diff
@@ -1 +1,54 @@
-/* add styles here */
+.page {
+  margin: 0;
+  width: 140px;
+  height: 160px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.main {
+  flex-wrap: wrap;
+  width: 96px;
+  height: 100%;
+  
+  justify-content: center;
+  display: block;
```

Comment:
The `.main` class uses `display: block;`, which conflicts with the flexbox properties like `flex-wrap`. Consider using `display: flex;` to maintain consistency with the flexbox layout.